style-guide/styles/wds-components/_global-navigation.scss

```diff
@@ -377,6 +378,21 @@ $hover-transition-duration: 300ms !default;
 		display: none;
 	}
 
+	&__right-slot {
+		max-width: 100px;
+		display: none;
+		height: 100%;
+		position: absolute;
+		right: 0;
+		top: 0;
+		border-left: 1px solid $color-blue-gray;
+
+		@media (min-width: 1280px) {
```

Comment:
arbitrary but very well chosen number - it just fits only when your screen is 1280px+,
I can add a comment about it